TheFourFifths/consus
# Email system tff 175

### Summary

System to send emails to people.

### Checklist


### How to Review

1. Pull
2. Make sure tests work.
3. Make sure code makes sense. 

### Links

- [TFF-175](https://msoese.atlassian.net/browse/TFF-175)
